src/components/Table/components/TableCell/components/RowPositionCell/rowPositionCell.tsx

```diff
@@ -0,0 +1,16 @@
+import { Typography } from "@mui/material";
+import { CellContext, RowData } from "@tanstack/react-table";
+import React from "react";
+import { BaseComponentProps } from "../../../../../types";
+import { TYPOGRAPHY_PROPS } from "./constants";
+
+export const RowPositionCell = <TData extends RowData, TValue>({
+  className,
+  ...cellContext
+}: BaseComponentProps & CellContext<TData, TValue>): JSX.Element => {
+  return (
+    <Typography {...TYPOGRAPHY_PROPS} className={className} component="div">
+      {cellContext.row.getRowPosition()}
```

Comment:
`RowPosition` is now a custom feature, and we now have `getRowPostiion` on the `cellContext.row`.